oc_portal_commons/views_utils.py

```diff
@@ -3,10 +3,10 @@
 from django.shortcuts import redirect
 from django.views import generic
 from django.utils.safestring import mark_safe
-from dlmanager.models import Delivery, JiraProjects, JiraInstances
+from oc_delivery_apps.dlmanager.models import Delivery, JiraProjects, JiraInstances
 
 
-from portal_commons.forms import DeliveryFormFilter
+from oc_portal_commons.forms import DeliveryFormFilter
```

Comment:
Just a note for future: better use relative import if we want to use something from the same package or its sub-packages.
```
from .forms import DeliverFormFilter
```
With such a thing we well not face the problems with module/package re-naming and re-location. Especially if we have got two versions installed in different locations simultaneously during development.